packages/@okta/vuepress-theme-prose/components/SearchBar.vue

```diff
@@ -1,10 +1,12 @@
 <template>
   <div class="search--wrapper">
     <div class="SearchBox" data-search-bar data-pipeline="developer-okta-com">
-      <div class="CoveoOmnibox"></div>
+      <div class="search--form">
+        <div class="CoveoOmnibox"></div>
+        <div class="CoveoSearchButton">Search</div>
+      </div>
       <div class="CoveoAnalytics"></div>
     </div>
-    <img class="input--icon" src="/img/icons/search.svg" />
```

Comment:
If this is removed, can we delete the `search.svg`?

Reply:
There is a reference to this image from _top-bar.scss